classes/class.ilObjMumieTask.php

```diff
@@ -553,7 +553,7 @@ public function setPrivateGradepool($private_gradepool)
 
     public function isGradepoolSet()
     {
-        return !is_null($this->private_gradepool);
+        return !((int)$this->private_gradepool === -1);
```

Comment:
I don't think typecasting is really necessary

Reply:
after testing typecasting is necessary otherwise the choice for a gradepool setting is disabled on creation